### DISCUSSION TITLE
Use Mergify For SAB

### DISCUSSION THREAD

**Discussion by Shluffy** [ScratchAddons/ScratchAddons#3084] [Ideas] [answered]
We can user mergify.io for sab when is made, or make a condition

**Comment by cobaltt7** [comment, marked as answer]
no. if we do that we might as well make him just commit directly

**Reply by Shluffy** [reply]
okay. Why not make sab commit it directly

**Reply by cobaltt7** [reply]
because we want to review it

we don't want to put unreleased addon's data on the website

or like a few weeks ago when I broke the l10n system

**Comment by CST1229** [comment]
What is SAB?
EDIT: Oh wait, probably ```scratchaddons-bot```

**Reply by cobaltt7** [reply]
[@scratchaddons-bot[bot]](https://github.com/apps/scratchaddons-bot)
